src/elements/TableCell/TableCell.js

```diff
@@ -185,7 +186,7 @@ const TableCell = ({
           className="text_small"
           template={<TextTooltipTemplate text={data.tooltip || data.value} />}
         >
-          {data.value}
+          {data.headerId === 'size' ? prettyBytes(item.size) : data.value}
```

Comment:
You can't add this method here. You need to add it to each place where the content is generated

Reply:
@mavdryk the prettyBytes i think item.size is not generated, it's looks like the item.size is raw data.

 The function is used in "PreviewModal.js" file. 

i can add
`  <div className="item-data">
                size:
                {artifact.ui.size
                  ? artifact.ui.size
                  : typeof artifact.size === 'string'
                  ? artifact.size
                  : prettyBytes(artifact.size)}
              </div>`

Reply:
for artifacts we use createArtifactsContent function where you should remove the old method convertBytes and instead use prettyBytes. So there won't be need to add prettyBytes in every place in the table where we'll use size. For now, you're still using convertBytes which returns wrong size value and then you pass this wrong value to the prettyBytes